### PR TITLE
Add audio and colliders

### PR DESCRIPTION
**Pull request by ritijain**
## Overview
Polishing InteractableExamples scene

## Changes
- Fixes: #6632 .
- Added AudioReceiver to all Interactable Buttons
- Added BoxCollider to Quads that serve as backing for buttons 
- Updated the speech profile asset for the interactable scene to enable Toggle Profiler and Toggle Diagnostics commands

### CODE REVIEW

**Comment by davidkline-ms** [comment]
@ritijain , can you please resolve the merge conflict? we should then be able to merge. thanks!

File: Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Scenes/InteractablesExamples.unity [hunk 1]
```diff
@@ -4306,58 +4296,16 @@ PrefabInstance:
...
-      objectReference: {fileID: 0}
-    - target: {fileID: 114818926546564510, guid: 02c524b22137b5449904f5395141cc73,
-        type: 3}
-      propertyPath: Events.Array.data[0].Settings.Array.data[0].Name
-      value: OnFocusOff
-      objectReference: {fileID: 0}
-    - target: {fileID: 114818926546564510, guid: 02c524b22137b5449904f5395141cc73,
-        type: 3}
-      propertyPath: Events.Array.data[0].Settings.Array.data[0].Tooltip
-      value: Focus has left the object
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Scenes/InteractablesExamples.unity]: Are these tooltip text settings no longer needed? Or are they represented elsewhere now?

Comment (ritijain) [reply Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Scenes/InteractablesExamples.unity]: yep. They are no longer needed.

**Pull request merged**